src/elements/category-title/src/index.tsx

```diff
@@ -0,0 +1,37 @@
+/** @jsx jsx */
+
+import * as React from 'react'
+import { IntrinsicSxElements, jsx, Styled, useThemeUI } from 'theme-ui'
+
+interface ListProps extends React.OlHTMLAttributes<HTMLUListElement> {
+  level: 1 | 2 | 3 | 4 | 5 | 6
+}
+
+const CategoryTitle: React.FC<ListProps> = ({ level = 2, children }) => {
+  const { theme }: any = useThemeUI()
+
+  const headingElement = `h${level}` as keyof IntrinsicSxElements
+  const Heading = Styled[headingElement]
+
+  return (
+    <div
+      sx={{
+        paddingX: ['sm', 'md'],
+        paddingY: ['md', 'lg'],
+        ...(theme.categoryTitle && theme.categoryTitle.main)
+      }}
+    >
+      <Heading
```

Comment:
I would expect that this could even be a p tag as it is a category, not a heading for a particular bit of content

Reply:
In figma it's called a category title - I can't see any situations where you wouldn't want a title to be a heading

Reply:
done 👍